edx_proctoring/templates/emails/proctoring_attempt_status_email.html

```diff
@@ -0,0 +1,9 @@
+{% load i18n %}
+
+{% blocktrans %}
+
+This email is to let you know that the status of your proctoring session review for {{ exam_name }} in
+<a href="{{ course_url }}">{{ course_name }} </a> is {{ status }}. If you have any questions about proctoring,
+contact edX support at info@edx.org.
```

Comment:
Please don't hard code 'edX' and the info@edx.org here since other Open edX users will be using this software.

Instead of 'edX', pass in as a context field settings.PLATFORM_NAME. I'm not sure where to get info@edx.org from configuration, but I'm sure there's an existing configuration setting. I can look if you have trouble finding it.